src/main/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPlugin.groovy

```diff
@@ -43,7 +43,9 @@ class DockerRemoteApiPlugin implements Plugin<Project> {
         // if no repositories were defined fallback to buildscript
         // repositories to resolve dependencies as a last resort
         if (project.repositories.size() == 0) {
```

Comment:
Planned on doing another release this weekend as docker-java put out another release this morning fixing the "start container with post data" issue. I'll sneak this in.